frontend/src/util/boardBuilder.test.ts

```diff
@@ -35,27 +35,27 @@ test(
   'Builder adds walls appropriately',
   () => {
     let board = new BoardBuilder()
-      .withWall(Wall.North,{ row: 1, column: 0 })
-      .withWall(Wall.East, { row: 1, column: 0 })
+      .withWall(Direction.North,{ row: 1, column: 0 })
+      .withWall(Direction.East, { row: 1, column: 0 })
       .build();
 
-    expect(board.cells[1]?.[0]?.walls).toStrictEqual([Wall.North, Wall.East]);
-    expect(board.cells[0]?.[0]?.walls).toStrictEqual([Wall.South]);
-    expect(board.cells[1]?.[1]?.walls).toStrictEqual([Wall.West]);
+    expect(board.cells[1]?.[0]?.walls).toStrictEqual([Direction.North, Direction.East]);
+    expect(board.cells[0]?.[0]?.walls).toStrictEqual([Direction.South]);
+    expect(board.cells[1]?.[1]?.walls).toStrictEqual([Direction.West]);
   }
 )
 
 test(
   "Builder doesn't crash when border walls are added",
   () => {
     let board = new BoardBuilder()
-      .withWall(Wall.North, { row: 0, column: 0 })
-      .withWall(Wall.West, { row: 0, column: 0 })
-      .withWall(Wall.East, { row: 15, column: 15 })
-      .withWall(Wall.South, { row: 15, column: 15 })
+      .withWall(Direction.North, { row: 0, column: 0 })
+      .withWall(Direction.West, { row: 0, column: 0 })
+      .withWall(Direction.East, { row: 15, column: 15 })
+      .withWall(Direction.South, { row: 15, column: 15 })
```

Comment:
nit:

Ideally the Wall -> Direction changes would be moved to their own pull request to make the functional changes of this pull request less ambiguous.

I know we talked about it last week and I said not to worry about it, just something to think about :)